tools/corto/src/cortotool.c

```diff
@@ -168,6 +168,16 @@ int main(int argc, char* argv[]) {
                     goto error;
                 }
                 break;
+            } else if (!strcmp(argv[i], "genrakefile")) {
+                if (cortotool_genrakefile(argc-1, &argv[i])) {
```

Comment:
What is the difference between "genrakefile" and "rakefile"? Please use more descriptive names, especially if these commands are going to be publicly visible to users.